task/rpms-signature-scan/0.2/rpms-signature-scan.yaml

```diff
@@ -48,7 +48,7 @@ spec:
         optional: true
   steps:
     - name: rpms-signature-scan
-      image: quay.io/konflux-ci/tools@sha256:f6b2dc74f37388a199eb258257d14e48359fda2757eee75258216b5711624592
+      image: quay.io/konflux-ci/tools@sha256:7b65b7993267183f800dbb8a631cf049abab0c0b91fe25f1708000fa7a702a26
```

Comment:
And this is not the same digest as in the 0.1 version of the task. Which one is the correct expected one?

Reply:
The one in 0.2 is correct. 0.1 is excluded from the renovate configs upstream so it's not getting updates.